src/views/Wallet/components/LeftNavigation/components/CoinMenu/index.js

```diff
@@ -75,12 +93,41 @@ class CoinMenu extends PureComponent<Props> {
             });
     }
 
+    isTopMenuEmpty() {
+        const numberOfVisibleNetworks = this.props.localStorage.config.networks
+            .filter(item => !item.isHidden) // hide coins globally in config
+            .filter(item => this.props.wallet.hiddenCoins.includes(item.shortcut));
+
+        return numberOfVisibleNetworks.length <= 0;
+    }
+
+    isBottomMenuEmpty() {
+        const { hiddenCoins } = this.props.wallet;
+        const numberOfVisibleNetworks = coins
+            .filter(item => !item.isHidden)
+            .filter(item => hiddenCoins.includes(item.id));
+
+        return numberOfVisibleNetworks.length <= 0;
+    }
+
+    isMenuEmpty() {
+        return this.isTopMenuEmpty() && this.isBottomMenuEmpty();
+    }
+
     render() {
+        const { hiddenCoins } = this.props.wallet;
         const { config } = this.props.localStorage;
         return (
             <Wrapper data-test="Main__page__coin__menu">
+                {this.isMenuEmpty() && (
+                    <Empty>
+                        <StyledLinkEmpty to="/settings">Select a coin</StyledLinkEmpty>
+                        <Gray> in application settings</Gray>
+                    </Empty>
+                )}
                 {config.networks
-                    .filter(item => !item.isHidden)
+                    .filter(item => !item.isHidden) // hide coins globally in config
+                    .filter(item => hiddenCoins.includes(item.shortcut)) // hide coins by user settings
```

Comment:
I think you are actually showing coins that are in `hiddenCoins` array. I suppose it should the opposite. It is kinda confusing.

Anyway this is what I got even without messing with the settings. I think we should keep all coins visible by default.

<img width="323" alt="Screenshot 2019-04-22 at 14 01 25" src="https://user-images.githubusercontent.com/6961901/56499819-65963f80-6508-11e9-95df-e9ab71ad4af0.png">